README.md

```diff
@@ -1,52 +1,92 @@
-
 # torus-direct-react-native-sdk
 
-## Getting started
+Fully white-labelled UI/UX paired up to Torus PKI and auth for React Native.
 
-`$ npm install @toruslabs/torus-direct-react-native-sdk --save`
+**Important**
 
-Please refer to the native sdks for specific configuration.
+This SDK requires native modules and is designed to bring native experience to your React Native app without you doing any work.
 
-- [Android Sdk](https://github.com/torusresearch/torus-direct-android-sdk)
-- [iOS Sdk](https://github.com/torusresearch/torus-direct-swift-sdk)
+If you've already implemented a native auth experience or prefer to use plain JS React Native, you can use [torus-direct-web-sdk](https://github.com/torusresearch/torus-direct-web-sdk), which won't require native modules and can plug into your login system via `getTorusKey` and `getAggregateTorusKey` function.
 
-More info can be found on docs [here](https://docs.tor.us/direct-auth/redirects-and-service-workers#android-and-react-native-android)
+## Getting Started
 
+```bash
+npm i --save @toruslabs/torus-direct-react-native-sdk
+```
 
-### Mostly automatic installation
+Please refer to the native SDKs for platform-specific configuration.
 
-`$ react-native link torus-direct-react-native-sdk`
+- [Android SDK](https://github.com/torusresearch/torus-direct-android-sdk)
+- [iOS SDK](https://github.com/torusresearch/torus-direct-swift-sdk)
 
-### Manual installation
+### Automatic installation
 
+```bash
+react-native link torus-direct-react-native-sdk
```

Comment:
Automatic linking should work currently. https://reactnative.dev/docs/linking-libraries-ios